contrib/log_correlation/stackdriver/src/main/java/io/opencensus/contrib/logcorrelation/stackdriver/OpenCensusTraceLoggingEnhancer.java

```diff
@@ -36,13 +37,20 @@
   private static final String SAMPLED_LABEL_KEY = "sampled";
   private static final SpanSelection DEFAULT_SPAN_SELECTION = SpanSelection.ALL_SPANS;
 
+  /**
+   * Name of the property that overrides the default Stackdriver Project ID. The value is {@value}.
```

Comment:
Just to be super clear, could you please say that the "default" is actually fetched from the environment / is based on the project id that the app is running as. I realize the comment on the constructor repeats this but it's worth repeating.

Reply:
I tried to make the wording more consistent between the property name and constructor, and I referenced the method that returns the default cloud project ID in both.  I couldn't add a direct link to the `ServiceOptions.getDefaultProjectId()` Javadoc, though, since it's in another library.